sdk/core/ci.yml

```diff
@@ -72,8 +72,5 @@ extends:
       - name: azure-core-tracing-opentelemetry
         groupId: com.azure
         safeName: azurecoretracingopentelemetry
-    AdditionalModules:
-      - name: perf-test-core
-        groupId: com.azure
     MatrixReplace:
```

Comment:
Out of curiosity. I can see the output from [core-perf-test](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/core/azure-core-perf/src/test/java/com/azure/core/perf/CorePerfIntegrationTest.java#L29-L53) in the CI test report.
What mechanism makes sure that perf-test-xxx modules get compiled and tested?